packages/atomic/src/components/commerce/product-template-components/atomic-product-price/atomic-product-price.tsx

```diff
@@ -0,0 +1,61 @@
+import {Product} from '@coveo/headless/commerce';
+import {Component, h, Prop} from '@stencil/core';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {CommerceBindings} from '../../atomic-commerce-interface/atomic-commerce-interface';
+import {ProductContext} from '../product-template-decorators';
+
+/**
+ * @internal
+ * The `atomic-product-price` component renders the price of a product.
+ */
+@Component({
+  tag: 'atomic-product-price',
+  shadow: false,
+})
+export class AtomicProductPrice
+  implements InitializableComponent<CommerceBindings>
+{
+  @InitializeBindings() public bindings!: CommerceBindings;
+  public error!: Error;
+
+  @ProductContext() private product!: Product;
+
+  /**
+   * The currency to use in currency formatting. Possible values are the ISO 4217 currency codes, such as "USD" for the US dollar, "EUR" for the euro, or "CNY" for the Chinese RMB.
+   * See the current [currency & funds code list](https://www.six-group.com/en/products-services/financial-information/data-standards.html#scrollTo=maintenance-agency).
+   */
+  @Prop({reflect: true}) public currency!: string;
+
+  public render() {
+    const hasPromotionalPrice =
+      this.product?.ec_promo_price !== undefined &&
+      this.product?.ec_price !== undefined &&
+      this.product?.ec_promo_price < this.product?.ec_price;
+
+    return (
+      <div>
+        <atomic-product-number
+          class={`mx-1 ${hasPromotionalPrice && 'text-error'}`}
+          field={hasPromotionalPrice ? 'ec_promo_price' : 'ec_price'}
+        >
+          <atomic-format-currency
+            currency={this.currency}
+          ></atomic-format-currency>
```

Comment:
This is technically part of the search folder, but has nothing specific to search. Should I create an equivalent in the commerce folder, or would it be better to move it to the common folder?

One detail to note is that while the atomic-format-xxx components don't refer to "result", their storybook examples do.

Reply:
🤔 Prolly commons, all the `<atomic-format-*>` should move to commons even I reckon.

Reply:
out of scope of this PR tho :)

Reply:
Confirmed should be moved.